deploy/openshift/deployment-validation-operator-olm-with-polling.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: deployment-validation-operator
+objects:
+- apiVersion: operators.coreos.com/v1alpha1
+  kind: CatalogSource
+  metadata:
+    name: ${SOURCE}
+  spec:
+    sourceType: grpc
+    image: ${IMAGE}:${IMAGE_TAG}
+    displayName: Deployment Validation Operator
+    publisher: ${CATALOG_PUBLISHER}
+    updateStrategy:
+      registryPoll:
+        interval: ${REGISTRY_POLLING_INTERVAL}
+- apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: deployment-validation-operator
+  spec:
+    channel: ${CHANNEL}
+    name: deployment-validation-operator
+    source: ${SOURCE}
+    sourceNamespace: ${SOURCE_NAMESPACE}
+    config:
+      env:
+      - name: NAMESPACE_IGNORE_PATTERN
+        value: ${NAMESPACE_IGNORE_PATTERN}
+- apiVersion: operators.coreos.com/v1
+  kind: OperatorGroup
+  metadata:
+    name: deployment-validation-operator
+    namespace: ${NAMESPACE}
+  spec:
+    targetNamespaces:
+    - ${NAMESPACE}
+parameters:
+- name: IMAGE
+  value: quay.io/operatorhubio/catalog
+  displayName: Catalog image with the deployment-validation-operator
+  description: the image of a catalog to deploy that contains the deployment-validation-operator
+  required: true
+- name: IMAGE_TAG
+  value: "latest"
+  displayName: deployment-validation-operator catalog version
+  description: the version of the deployment-validation-operator catalog to deploy
+  required: true
+- name: NAMESPACE
+  value: "deployment-validation-operator"
+  displayName: namespace to deploy into
+  description: the namespace to deploy into
+  required: true
+- name: CHANNEL
+  value: "alpha"
+  displayName: OLM subscription channel
+  description: OLM subscription channel
+  required: true
+- name: NAMESPACE_IGNORE_PATTERN
+  value: ""
+  displayName: Namespace ignore pattern
+  description: A golang regex to ignore checks from matching patterns
+- name: SOURCE
+  value: "operatorhubio-catalog"
+  displayName: Operator catalog source
+  description: Catalog source that provides the operator
+- name: SOURCE_NAMESPACE
+  value: "deployment-validation-operator"
+  displayName: Catalog namespace source
+  description: Namespace containing the catalog source
+- name: CATALOG_PUBLISHER
+  value: "OperatorHub.io"
+  displayName: Publisher of the catalog
+  description: The name of the publisher of the catalog
+- name: REGISTRY_POLLING_INTERVAL
+  value: "15m"
```

Comment:
I would default this to 24h.  Every time this runs it consumes cluster resources, and atm it's not clear to me we would need to be this instantaneous for updates.

Reply:
Now set to `24h`